### PR TITLE
Add Braintree block to the Sales Menu page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request:
- Add Braintree block to the Sales Menu page
- adds Braintree page to the storefront

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages
https://docs.magento.com/user-guide/sales.html
https://docs.magento.com/user-guide/payment/braintree.html

### CODE REVIEW

File: src/sales/sales-menu.md [hunk 1]
```diff
@@ -47,6 +47,9 @@ A [billing agreement]({% link payment/paypal-billing-agreements.md %}) is simila
 
 The [Transactions]({% link sales/transactions.md %}) page lists all payment activity that has taken place between your store and all payment systems, and provides access to more detailed information.
 
+### Braintree Virtual Terminal
+On the Braintree Virtual Terminal page, the admin can accept the payment for the selected amount. To available the terminal feature customer should configure basic [Braintree settings]({% link payment/braintree.md %}). Braintree offers a fully customizable checkout experience with fraud detection and PayPal integration.
```
Comment (leslietilling) [inline src/sales/sales-menu.md:51]:
```suggestion
On the Braintree Virtual Terminal page, an Admin user can accept the payment for the selected amount. To make  the terminal feature available, a merchant should configure basic [Braintree settings]({% link payment/braintree.md %}). Braintree offers a fully customizable checkout experience with fraud detection and PayPal integration.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Minor edit

File: src/_data/toc/sales.yml [hunk 2]
```diff
@@ -255,6 +255,8 @@ pages:
       url: "/payment/zero-subtotal-checkout.html"
   - label: Fraud Protection
     url: "/sales/fraud-protection.html"
+  - label: Braintree
```
Comment (leslietilling) [inline src/_data/toc/sales.yml:258]: If this page is already in a toc file, it cannot be used a second time (causes problems with nav switching)

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**